gen3/neural-networks/generic-example/README.md

```diff
@@ -8,42 +8,72 @@ Moreover, you need to prepare a **Python 3.10** environment with the following p
+The example can be ran in this mode as:
 ```bash
 python3 main.py \
-    --model_slug ... \
-    --device ... \
-    --annotation_mode ... \
-    --fps_limit ... \
-    --media ...
+    --model <Model> \
+    --device <Device> \
+    --annotation_mode <Mode> \
+    --fps_limit <FPS> \
+    --media <Media>
 ```
 
-The relevant arguments:
-- **--model_slug**: A unique HubAI identifier of the model;
-- **--device** [OPTIONAL]: DeviceID or IP of the camera to connect to.
+Relevant arguments:
+- `<Model>`: A unique HubAI identifier of the model;
+- `<Device>` [OPTIONAL]: DeviceID or IP of the camera to connect to.
 By default, the first locally available device is used;
-- **--annotation_mode** [OPTIONAL]: Annotation mode. Set to 'segmentation' to overlay segmentation masks over the model inputs, or 'segmentation_with_annotation' to visualize the additional annotations. Leave empty to use the default visualization.
-- **--fps_limit** [OPTIONAL]: The upper limit for camera captures in frames per second (FPS).
+- `<Mode>` [OPTIONAL]: Annotation mode. Set to 'segmentation' to overlay segmentation masks over the model inputs, or 'segmentation_with_annotation' to visualize the additional annotations. Leave empty to use the default visualization.
+- `<FPS>` [OPTIONAL]: The upper limit for camera captures in frames per second (FPS).
 The limit is not used when infering on media.
 By default, the FPS is not limited.
 If using OAK-D Lite, make sure to set it under 28.5;
-- **--media** [OPTIONAL]: Path to the media file to be used as input. 
+- `<Media>` [OPTIONAL]: Path to the media file to be used as input. 
 Currently, only video files are supported but we plan to add support for more formats (e.g. images) in the future.
 By default, camera input is used;
 
 Running the script downloads the model, creates a DepthAI pipeline, infers on camera input or the provided media, and display the results by **DepthAI visualizer**
 The latter runs in the browser at `http://localhost:8082`.
 In case of a different client, replace `localhost` with the correct hostname.
 
-## Example
-To try it out, let's run a simple YOLOv6 object detection model on your camera input.
+### Example
+To try it out, let's run a simple YOLOv6 object detection model on your camera input:
 ```bash
 python3 main.py \
-    --model_slug luxonis/yolov6-nano:r2-coco-512x288
+    --model luxonis/yolov6-nano:r2-coco-512x288
 ```
+
+## Standalone Mode
+Running the example in the [Standalone mode](https://rvc4.docs.luxonis.com/software/depthai/standalone/), app runs entirely on the device.
+To run the example in this mode, first install the [oakctl](https://rvc4.docs.luxonis.com/software/tools/oakctl/) command-line tool (enables host-device interaction) as:
+```bash
+bash -c "$(curl -fsSL https://oakctl-releases.luxonis.com/oakctl-installer.sh)"
+```
+and run the example using the `main_standalone.sh` script:
+```bash
+bash main_standalone.sh \
```

Comment:
Let's rename to `run_standalone.sh` or if maybe have better name for it

Reply:
Added in [45e4f16](https://github.com/luxonis/depthai-experiments/pull/564/commits/45e4f166dcc2b3095b228d978aa91cc63542ce28).